tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -10505,6 +10534,11 @@ void iop_export_public_key(
                                                 &output_len);
     TEST_EQUAL(status, PSA_ERROR_BAD_STATE);
 
+    PSA_ASSERT(psa_export_public_key_iop_abort(&export_key_operation));
+
+    num_ops = psa_export_public_key_iop_get_num_ops(&export_key_operation);
+    TEST_EQUAL(num_ops, 0);
+
```

Comment:
In the sequence
```
/*first operation*/
setup()
complete() until SUCCESS
/*second operation using the same operation object*/
setup()
complete()
get_num_ops()
```

the call to `get_num_ops` should only get the op count from the second operation. Is this tested anywhere?

Reply:
Added a test for it.